pdp/static/js/pdp_controls.js

```diff
@@ -308,6 +310,11 @@ RasterDownloadLink.prototype = {
 
     // Take the URL template and substitute each of the desired state variables
     getUrl: function () {
+        //if this portal requies the user to select an area, don't generate a URL
+        //unless they've made a selection.
+        if(this.requireArea && !this.areaSpecified) {
```

Comment:
I'm not a big fan of this solution. I'd rather that we omit the i/j parameters by default (which is a valid OPeNDAP request) until the user selects a rectangle.

This map require some changes to the `RasterDownloadLink` in `pdp_controls.js` or simply initializing the `RasterDownloadLink` with empty coordinates.